dependencies.gradle

```diff
@@ -1,13 +1,12 @@
 ext {
     // -- PLUGINS
     spotlessVersion = '6.25.0'
-    benmanesVersion = '0.51.0'
 
     // -- DEPENDENCIES
     gdxVersion = '1.12.1'
     aiVersion = '1.8.2'
     junitVersion = '4.13.2'
-    mockitoVersion = '5.11.0'
+    mockitoVersion = '5.10.0'
```

Comment:
```suggestion
    mockitoVersion = '5.11.0'
```

Reply:
Das ist nur zu Testzwecken drin, ob der Dependabot noch anspringt.

Reply:
Ich würde auch sagen, das finden wir nur durch einen Versuch heraus.

Reply:
Ja, tut er. Hatte den Branch in ein privates Repo gepusht, da war der Bot sofort da.